src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age = 0, sex, isMarried, partnerName } = person;
+  const personPartnerMessage = (sex === 'm')
+    ? `${partnerName} is my wife`
+    : `${partnerName} is my husband`;
+
+  return (
+
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age
+        ? <p className="Person__age">{`I am ${age}`}</p>
+        : null}
```

Comment:
instead of using ternary operator, just write `&&`

in this case you're gonna have less code